app/controllers/idv/account_verified_cta_visited_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Idv
+  class AccountVerifiedCtaVisitedControllerError < ActionController::ActionControllerError; end
+
+  class AccountVerifiedCtaVisitedController < ApplicationController
+    before_action :disable_caching
+
+    rescue_from AccountVerifiedCtaVisitedControllerError, with: :render_not_acceptable
+
+    def show
+      if redirect_url.present?
+        redirect_to(redirect_url, allow_other_host: true)
+        analytics.idv_account_verified_cta_visited(**valid_params)
+      else
+        raise AccountVerifiedCtaVisitedControllerError,
+              "No destination URL found for issuer: #{issuer}."
+      end
+    end
+
+    private
+
+    def redirect_url
+      sp_return_url_resolver&.return_to_sp_url
+    end
+
+    def issuer
+      valid_params[:issuer]
+    end
+
+    def campaign_id
+      valid_params[:campaign_id]
+    end
+
+    def valid_params
+      @valid_params ||=
+        params.permit(:campaign_id, :issuer).to_h.symbolize_keys
```

Comment:
```suggestion
        params.permit(:campaign_id, :issuer)
```

Generally params should not be converted to a hash since the parameter validation functionality is lost ([docs](https://api.rubyonrails.org/classes/ActionController/StrongParameters.html)). A mass assignment vulnerability isn't necessarily relevant here, but I think we ideally use the parameters as a good practice.

Reply:
I missed that it was used for the analytics, but I think we could expand and pass the values explicitly since there's only two